src/styles/_catalog.scss

```diff
@@ -1,22 +1,32 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  align-items: center;
+
+  grid-template-columns: repeat(4, 200px);
+  grid-auto-flow: row;
```

Comment:
row is the default value. in my opinion it can be removed

Reply:
Agree